python/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine
+fiFROM alpine
```

Comment:
Why fiFROM? Is this a keyword in docker?

Reply:
Sorry...this was accidentally done by my cat, I didn't intend to make this change. >A<